scripts/treim.lic

```diff
@@ -602,7 +502,7 @@ elsif ((variable[1] == nil || variable[1].downcase == "help") && UserVars.treim[
 	
   Running Commands
      ;send reimcount   - shows current group leader, group count, and previos group count
-     whisper ooc group clearcheck - if group members are running script,
+     whisper ooc group checkclear - if group members are running script,

```

Comment:
The OOC command didn't change from clearcheck that the script looks for. So may want to revert this for instructions so people don't get confused.